src/view/utils/fetchFromPlatform.jsx

```diff
@@ -22,8 +22,21 @@ const PLATFORM_HOST_STAGING = "https://platform-stage.adobe.io";
 const ERROR_CODE_INVALID_ACCESS_TOKEN = "401013";
 const ERROR_CODE_USER_REGION_MISSING = "403027";
 
-const ERROR_NO_AEP_ACCESS =
-  "Your organization is not provisioned for Adobe Experience Platform. Please contact your organization administrator.";
+const ERROR_NO_AEP_ACCESS = (
+  <>
+    You or your organization is not currently provisioned for Adobe Data
+    Collection. To request personal access contact your organization
+    administrator, or to request the necessary access for your organization
+    follow the instructions{" "}
+    <a
```

Comment:
@jonsnyder I suggest we remove the link to the instructions because that form will be removed soon because all customers will be auto provisioned.